Doc/library/stdtypes.rst

```diff
@@ -5162,6 +5162,14 @@ enables cleaner type hinting syntax compared to :data:`typing.Union`.
       def square(number: int | float) -> int | float:
           return number ** 2
 
+   .. note::
+
+      The ``|`` operand cannot be used at runtime to define unions where one or
+      more members is a forward reference. For example, ``int | "Foo"``, where
```

Comment:
```suggestion
      The ``|`` operand cannot be used at runtime to define unions that contain
      forward references. For example, ``int | "Foo"``, where
```
To avoid the awkwardness of "members is"

Reply:
Hmm, not sure. It feels weird to think of `UnionType` as a "container"; my current wording feels more precise to me. (And I don't personally find it especially awkward :)